### ISSUE TITLE
"Automatically launch when i turn my computer on" on Ubuntu doesn't work.

### ISSUE THREAD

**Issue by antoine-morisseau** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#2163]
**OS:**
Ubuntu 16.04

**Issue Descriptions:**
The application doesn't start at the opening of my session when i start my computer.

**Steps to Reproduce:**
Get a Ubuntu 16.04 LTS. Create a account. Install the application, activate the option, restart the computer and login.

I asked on gitter if anyone could reproduce on Ubuntu and the user @GreenTurtwig could reproduce the issue. (the discussion is here : https://gitter.im/MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-?at=5888711ce836bf70108bf7cc )

**Comment by MarshallOfSound** [comment]
@antoine-morisseau Is there a `.desktop` entry for GPMDP in `~/.config/autostart`

**Comment by antoine-morisseau** [comment]
Yep there is a `Google Play Music Desktop Player.desktop` file in the `~/.config/autostart folder`.
Here is all the values : 

> [Desktop Entry]
Type=Application
Vestion=1.0
Name=Google Play Music Desktop Player
Comment=Google Play Music Desktop Player startup script
Exec=/usr/share/google-play-music-desktop-player/Google Play Music Desktop Player
StartupNotify=false
Terminal=false

**Comment by MarshallOfSound** [comment]
I'm guessing it's because `/usr/share/google-play-music-desktop-player/Google Play Music Desktop Player` isn't escaped (there are spaces).  Can you try manually escaping that path and rebooting 👍

**Comment by antoine-morisseau** [comment]
yeah ! it start correctly now !
But it generate another issue about the icon on the unity bar. It's not the google may music but the google icon. i can attach a screenshot if necessary.

**Comment by MarshallOfSound** [comment]
@antoine-morisseau Probably because their is no Icon property in that desktop file.  I'll look into fixing this in `node-autostart` at some point 👍

**Comment by antoine-morisseau** [comment]
By the way, with now GPMDP starting at launch, could be a new option that launch minimized to tray when the application start at the computer turns on ?
